js/components/main.js

```diff
@@ -511,6 +511,10 @@ class Main extends ImmutableComponent {
 
     const onWindowMove = debounce(function (event) {
       const position = event.sender.getPosition()
+      // Windows may be placed off-screen
+      // See https://github.com/brave/browser-laptop/issues/3558
+      position[0] = Math.max(0, position[0])
```

Comment:
@bridiver I feel a little strange by clamping to 0 indiscriminately for all negative values too. That being said, if 0,0 is the top-left most position of the top-left most display, I can't think of a reason why a user would have their browser hanging off that end. _I can think of reasons why they may want their browser to over-hang bottom-right._

I'll check out the screen API, to see if there's a more elegant approach. The primary issue I'd like to resolve is the case where the browser window is further to the negative than it is wide, resulting in an unreachable UI.